app/views/course/quiz_questions/_card_body.html.erb

```diff
@@ -8,7 +8,7 @@
     <% else %>
       <h4>Not quite right</h4>
     <% end %>
-    <h5>You picked: <%= quiz_question.answer.content %></h5>
+    <h5>You picked: <%= quiz_question.answer.content.to_plain_text %></h5>
```

Comment:
Any reason for this? It will essentially mean that is the answer has HTML formatted things (strong tags, code tags, links, etc...) it will be ignored.

Reply:
@jamiesonreinhard

Reply:
I added it so that the answer would stay on the same line as "You Picked:". If there's a better way to do this, I can change it. I think it looks better this way. @caioertai

Reply:
Alright. I don't think it's relevant until we have an actual requirement for having HTML formatted answer content. So if it looks better now, I'm in.

Reply:
@caioertai I found another way of doing this by adding display: inline; to the .card .trix-content css class. I'm not sure if this will affect any other elements, though. Let me know if that's a better solution. If not I'll change it back to use the .to_plain_text.

Reply:
It will definitely affect other elements. :grimacing: 
The `trix-content` tag gets added to any content coming from the action text rich text fields. Which includes every section content.

You should revert this.

You could instead create a namespace wrapper for a different trix-content style. Like:
```css
.inline-trix-content {
  .trix-content {
    display: inline;
  }
}
```

Then use the `.inline-trix-content` before those content tags.

Reply:
Oh. I just noticed that you were changing this directly on cards only. My bad. Ignore parts of the last comment. I still stand by the namespace for trix content tho. It's a more general additive approach. It would still allow for card `trix-content` to be inline IF, and only IF, needed. And allow for it to be used in other places as well. There's a trix css file somewhere where I keep the overrides.

Reply:
Ok, got it. I fixed it using the namespace approach anyway, which you'll see in the last commit. I'm headed to work shortly so I should be done making changes to this and we can merge it when Ben has reviewed. Thanks for your help.